xpdacq/beamtime.py

```diff
@@ -168,6 +168,65 @@ def Tramp(dets, exposure, Tstart, Tstop, Tstep, *, md=None):
     yield from plan
 
 
+def T_list(dets, exposure, T_list):
+    """
+    Scan over the temperature controller in a list of user defined steps
+
+    Parameters
+    ----------
+    dets : list
+        list of objects represent instrument devices. In xpdAcq, it is
+        default to area detector.
+    exposure : float
+        total time of exposure in seconds for area detector
+    T_list : list
+        a list of temperature steps that you wish to scan over
+
+    Note
+    ----
+    area detector and temperature controller that would be triggered will
+    always be the one configured in global state. Please using following
+    commands:
+
+        >>> glbl.area_det
+        >>> glbl.temp_controller
+
+    To figure interrogate devices that are currently in use.
+    """
+
+    pe1c, = dets
+    # setting up area_detector and temp_controller
+    (num_frame, acq_time, computed_exposure) = _configure_pe1c(exposure)
+    motor = glbl.temp_controller
```

Comment:
why on earth are you using `motor` for the variable name here.  There is no motor in this problem.
Better to call it `t_controller` or something.